.gitignore

```diff
@@ -1,7 +1,2 @@
-# ignores all .md files
-.md
-
-# does not ignore the README.md file
-!README.md
-
-git rm --cached /path/to/test.md
+# ignores test.md file
+/test.md
```

Comment:
- [ ] Currently you don't have a test.md file. Please add the test.md file to your project. You can ignore that file so that it is not pushed into github just by the name of the file (test.md) in your `.gitignore` file. 🤓 

This is just for checking that you know how to do this in the future 👍